node/src/components/storage.rs

```diff
@@ -712,6 +751,37 @@ impl Storage {
             .transpose()
     }
 
+    pub fn get_highest_block_header_for_db_check(&self) -> Option<Block> {
```

Comment:
We can drop the `_for_db_check` suffix.

Reply:
Also, since this a public method please add a docstring.

Reply:
Added docstring